### ISSUE TITLE
Add setting for dash to be out of fixed resolution

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#1997]
### Is your feature request related to a problem? Please describe.

I want a setting that can set the dash to a fixed resolution so that your UI is consistent between desktop and VR plus if you could edit the resolution of the dash you can have some empty space on the left and right which could be useful

### Describe the solution you'd like

I would like to edit the resolution of the dash to be a fixed resolution between desktop and VR

### Describe alternatives you've considered

None to my knowledge accept a mod

### Requesters

#epicEaston197

**Comment by shiftyscales** [comment]
> UI is consistent between desktop and VR

Is there specific reason this is important, @epicEaston197? facets and things should scale dynamically to fill the available space.

Is something breaking that is causing you to make this request?

**Comment by 5H4D0W-X** [comment]
The online status facet can expand incorrectly when the dash extends in desktop mode. This could also functionally break some facets if they are placed in such a way that every facet in a row is at it's horizontal minimum size. I use the mod that prevents the dash from filling the screen in desktop and would really appreciate an option to set a fixed resolution.

**Comment by shiftyscales** [comment]
> This could also functionally break some facets if they are placed in such a way that every facet in a row is at it's horizontal minimum size.

Could, or does, @5H4D0W-X? Is that fact, or speculation.

> The online status facet can expand incorrectly when the dash extends in desktop mode.

This is why I'm asking. If things worked properly, it wouldn't be necessary to have a mod that enforces exact sizes.

**Comment by JackTheFoxOtter** [comment]
I agree. I do believe having the Dash in desktop mode respect the same aspect ratio as in VR is valuable and would look significantly better.

**Comment by ShrikeAlvaron** [comment]
> > UI is consistent between desktop and VR
> 
> Is there specific reason this is important, @epicEaston197? Facets and things should scale dynamically to fill the available space.
> 
> Is something breaking that is causing you to make this request?

I'm the one who kinda inspired this issue to be made and in my case I use the mod so that the inventory grid alignment is consistent between both VR and desktop, so that if I'm used to something being say the first item on the second row then it stay there no matter which mode I'm in.  I figure that when an inventory rework is done that particular issue could be addressed then though.

**Comment by epicEaston197** [comment]
On top of the above it's just kinda nice to be more aware of your surroundings when your dash is open

**Comment by art0007i** [comment]
@jae1911 info provided, issue closed :(

**Comment by jae1911** [comment]
Rah, sorry about that one.

**Comment by ko-tengu** [comment]
I think it is worth noting that this is especially impactful at ultrawide resolutions, causing the dashboard to become very stretched and spread out

<img width="1588" height="940" alt="Image" src="https://github.com/user-attachments/assets/a0fa72c2-b246-4191-8e82-194cd60697ff" />

<img width="1592" height="663" alt="Image" src="https://github.com/user-attachments/assets/a5260b51-ecf9-4285-9cd6-0aa4c8116d5a" />

<img width="1618" height="964" alt="Image" src="https://github.com/user-attachments/assets/a39f1a6f-6199-46b0-a664-cfee0b7a3c71" />

<img width="1590" height="660" alt="Image" src="https://github.com/user-attachments/assets/5bbbea0c-d9b2-4e3e-9265-782bd3a15b6b" />
